.pre-commit-config.yaml

```diff
@@ -82,7 +82,7 @@ repos:
     language_version: python3
 
 - repo: https://github.com/PyCQA/flake8.git
-  rev: 4.0.1
+  rev: 5.0.4
```

Comment:
@asottile does pre-commit.ci have a way to ignore a specific hook repo entry? There's 2 flake8 versions because one of the plugins doesn't yet support flake8 5, so it has a separate entry where it's being executed with flake8 4.

Reply:
Upstream issue: https://github.com/pre-commit-ci/issues/issues/140.

Reply:
Apparently, this is an old and rejected request: https://github.com/pre-commit-ci/issues/issues/83. It's not going to be implemented, but the comments and links there suggest that setting the autoupdate schedule to `quarterly` would make this bearable, so that's what I'm going to do.